core/templates/pages/splash-page/splash-page.component.ts

```diff
@@ -61,16 +64,78 @@ angular.module('oppia').component('splashPage', {
         return false;
       };
 
+      ctrl.onClickLearnMoreStudentsButton = function() {
+        $timeout(function() {
+          WindowRef.nativeWindow.location = '/learn/math';
+        }, 150);
```

Comment:
Why is this timeout needed here? Since the button is only used as a link should we maybe use `<a>` and change the appearance of it as recommended here https://stackoverflow.com/a/2906586/3688189

Reply:
Done, used the method in the link.